iothub/device/src/Pipeline/DefaultDelegatingHandler.cs

```diff
@@ -10,59 +10,51 @@ namespace Microsoft.Azure.Devices.Client.Transport
 {
     internal abstract class DefaultDelegatingHandler : IDelegatingHandler
     {
-        private volatile IDelegatingHandler _innerHandler;
+        private volatile IDelegatingHandler _nextHandler;
```

Comment:
It is private anyway, but wonder if 'next' does communicate 'delegating'.
Why don't we call it 'delegate' directly (delegating to this)?

Reply:
What is the delegating part? I've never understood that either.

It is a chain of handlers for an operation.